target/linux/rockchip/image/armv8.mk

```
@@ -293,6 +293,18 @@ define Device/mmbox_anas3035
 endef
 TARGET_DEVICES += mmbox_anas3035
 
+define Device/nlnet_xiguapi-v3
+  DEVICE_VENDOR := NLnet
+  DEVICE_MODEL := XiGuaPi V3
+  SOC := rk3568
+  DEVICE_DTS := rk3568-xgp-v3
+  UBOOT_DEVICE_NAME := xgp-rk3568
+  BOOT_FLOW := pine64-img
```

```suggestion
  $(Device/rk3568)
  DEVICE_VENDOR := NLnet
  DEVICE_MODEL := XiGuaPi V3
  DEVICE_DTS := rk3568-xgp-v3
  UBOOT_DEVICE_NAME := xgp-rk3568
```

DEVICE_DTS and UBOOT_DEVICE_NAME can be removed once you swtiched to standard naming